src/main/cpp/src/HashJni.cpp

```diff
@@ -21,6 +21,14 @@
 
 extern "C" {
 
+JNIEXPORT jint JNICALL Java_com_nvidia_spark_rapids_jni_Hash_getMaxNestedDepth(JNIEnv* env, jclass)
+{
+  try {
+    return spark_rapids_jni::MAX_NESTED_DEPTH;
+  }
+  CATCH_STD(env, 0);
```

Comment:
This is a trivial statement, nothing to try/catch.
```suggestion
    return spark_rapids_jni::MAX_NESTED_DEPTH;
```